indico/migrations/versions/20171117_1035_40d4569835b8_add_include_description_to_reminders.py

```diff
@@ -0,0 +1,27 @@
+"""Add include_description to reminders
+
+Revision ID: 40d4569835b8
+Revises: 
+Create Date: 2017-11-17 10:35:12.378432
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = '40d4569835b8'
+down_revision = None
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column('reminders', sa.Column('include_description', sa.Boolean(), nullable=False, default=False,
+	 server_default='false'), schema='events')
```

Comment:
wrong indentation; `default` is not affecting the database structure so it can be omitted here

Reply:
done